osclib/adi_command.py

```diff
@@ -89,19 +89,23 @@ def create_new_adi(self, wanted_requests, by_dp=False):
                     continue
 
                 non_ring_packages.append(target_package)
-                if by_dp is True:
-                    devel_project = self.get_devel_project(source_project, source_package)
-                    # try target pacakge in Factory
-                    # this is a bit against Leap development in case submissions is from Update,
-                    # or any other project than Factory
-                    if devel_project is None and self.api.project.startswith('openSUSE:'):
-                        devel_project = self.get_devel_project('openSUSE:Factory', target_package)
-                    if devel_project is not None:
-                        source_project = devel_project
-
-                if not source_project in non_ring_requests:
-                    non_ring_requests[source_project] = []
-                non_ring_requests[source_project].append(request_id)
+                if split:
+                    # request_id pretended to be a source project
+                    non_ring_requests[request_id] = [request_id]
```

Comment:
we are using as index `request_id`, but in by_dp we use `source_project`, is `source_project` a request id too? (sorry to be so lazy to check it for myself)